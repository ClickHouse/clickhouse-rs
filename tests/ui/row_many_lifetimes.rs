#[derive(clickhouse::Row)]
struct Row<'a, 'b> {
    a: &'a str,
    b: &'b str,
}

fn main() {}
