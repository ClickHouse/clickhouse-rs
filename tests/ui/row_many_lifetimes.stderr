error: `Row` cannot be derived for structs with multiple lifetimes
 --> tests/ui/row_many_lifetimes.rs:2:16
  |
2 | struct Row<'a, 'b> {
  |                ^^
